testing/src/cypress/e2e/04-api-institution/02-instition-create-update-location.cy.ts

```diff
@@ -2,19 +2,41 @@ import InstitutionHelperActions from "../../custom-command/institution/common-he
 import Authorization, {
   ClientId,
 } from "../../custom-command/common/authorization";
+import axios from "axios";
 
 const institutionHelperActions = new InstitutionHelperActions();
 
 const USERNAME = institutionHelperActions.getUserNameForApiTest();
 const PASSWORD = institutionHelperActions.getUserPasswordForApiTest();
 const TOKEN_URL = institutionHelperActions.getApiUrlForKeyCloakToken();
 
-function createOrUpdateInstitutionLocation(
+async function getUniqueInstitutionCode(token: string) {
+  let locationIds: string[] | PromiseLike<string[]> = [];
+  let newLocationId = institutionHelperActions.getRandomInstitutionCode();
+  const url = institutionHelperActions.getApiForGetAllInstitutionLocation();
+  const settings = {
+    headers: { Authorization: `Bearer ${token}` },
+  };
+  const response = await (await axios.get(url, settings)).data;
```

Comment:
I am not following why do we need to await on the get (which is ok) and then wait again?

Reply:
fixed it